cmd/utils/nodecmd/chaincmd.go

```diff
@@ -232,45 +212,53 @@ func writeFailLog(key int, err error) {
 	logger.Crit(msg, "err", err)
 }
 
-func checkGenesisAndFillDefaultIfNeeded(genesis *blockchain.Genesis) *blockchain.Genesis {
```

Comment:
`checkGenesisAndFillDefaultIfNeeded` is split into `genesis.Config.SetDefaults()` and `ValidateGenesisConfig(genesis)`.